### PR TITLE
Feature/gh 808 intro

### PR DESCRIPTION
**Pull request by DmitryBespalov**

### CODE REVIEW

File: SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/Welcome/OnboardingWelcomeViewController.swift [hunk 1]
```diff
@@ -6,34 +6,35 @@ import UIKit
...
     @IBOutlet weak var setupPasswordButton: StandardButton!
-    private weak var delegate: StartViewControllerDelegate!
+    private weak var delegate: OnboardingWelcomeViewControllerDelegate!
 
     private var preservedNavBarColor: UIColor!
     private var preservedTranslucent: Bool!
 
     private enum Strings {
-        static let description = LocalizedString("app_slogan", comment: "App slogan")
+        static let description = LocalizedString("ios_app_slogan", comment: "App slogan")
```
Comment (sche) [inline SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/Welcome/OnboardingWelcomeViewController.swift:24]: Do we have a different one?

File: SafeUIKit/SafeAppUI/ReusableDetachedControllers/BaseInputViewController.swift [hunk 2]
```diff
@@ -32,8 +39,19 @@ class BaseInputViewController: UIViewController, EventSubscriber {
...
         super.viewDidLoad()
-        headerLabel.attributedText = .header(from: headerText, style: headerStyle)
+        headerLabel.attributedText = NSAttributedString(string: headerText, style: OnboardingHeaderStyle())
         disableNextAction()
+        backgroundView?.isWhite = true
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if let nav = navigationController,
```
Comment (sche) [inline SafeUIKit/SafeAppUI/ReusableDetachedControllers/BaseInputViewController.swift]: Can you add some comment here?

File: SafeUIKit/SafeAppUI/ReusableDetachedControllers/RecoveryPhraseInputViewController.swift [hunk 3]
```diff
@@ -57,13 +57,23 @@ class RecoveryPhraseInputViewController: BaseInputViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        backgroundView.isWhite = true
         keyboardBehavior = KeyboardAvoidingBehavior(scrollView: scrollView)
+
+        phraseTextView.layer.cornerRadius = 10
```
Comment (sche) [inline SafeUIKit/SafeAppUI/ReusableDetachedControllers/RecoveryPhraseInputViewController.swift:63]: I think this style will be used in some places. Does it make sense to encapsulate it?

File: SafeUIKit/SafeAppUI/ReusableDetachedControllers/TwoFA/TwoFAViewController.swift [hunk 4]
```diff
@@ -124,15 +117,13 @@ public final class PairWithBrowserExtensionViewController: UIViewController {
...
-    override public func willMove(toParent parent: UIViewController?) {
-        guard let nav = parent as? UINavigationController,
-            nav.topViewController == self && nav.viewControllers.count > 1 else { return }
-        nav.viewControllers[nav.viewControllers.count - 2].navigationItem.backBarButtonItem = backButtonItem
-    }
-
     public override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         didCancel = false
+        if let nav = navigationController,
```
Comment (sche) [inline SafeUIKit/SafeAppUI/ReusableDetachedControllers/TwoFA/TwoFAViewController.swift]: Can you please add a comment here?

File: SafeUIKit/SafeAppUI/en.lproj/Localizable.strings [hunk 5]
```diff
@@ -175,6 +172,9 @@
    Insufficient funds header. */
 "insufficient_funds" = "Insufficient funds";
 
+/* App slogan */
+"ios_app_slogan" = "Enter the\ndecentralized web.";
```
Comment (sche) [inline SafeUIKit/SafeAppUI/en.lproj/Localizable.strings:176]: Ah, that's why :)

File: SafeUIKit/SafeUIKit/QRCodeScanner/ScannerViewController.swift [hunk 6]
```diff
@@ -68,6 +69,7 @@ class ScannerViewController: UIViewController {
         codeReaderVC.didMove(toParent: self)
 
         closeButton.accessibilityLabel = LocalizedString("close", comment: "Close button on camera")
+        buttonEffectView.mask = CircleMask(frame: buttonEffectView.bounds)
```
Comment (sche) [inline SafeUIKit/SafeUIKit/QRCodeScanner/ScannerViewController.swift:72]: nice :)

File: SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/Welcome/OnboardingWelcomeViewController.swift (hunk 1 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/MainFlow/OnboardingFlow/Welcome/OnboardingWelcomeViewController.swift:24]: this is with the '\n' in the middle of the phrase

File: SafeUIKit/SafeAppUI/ReusableDetachedControllers/BaseInputViewController.swift (hunk 2 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/ReusableDetachedControllers/BaseInputViewController.swift]: yes

File: SafeUIKit/SafeAppUI/ReusableDetachedControllers/RecoveryPhraseInputViewController.swift (hunk 3 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/ReusableDetachedControllers/RecoveryPhraseInputViewController.swift:63]: no, this view controller is reused as a whole, the text view does not appear anywhere else (so far).

File: SafeUIKit/SafeAppUI/ReusableDetachedControllers/TwoFA/TwoFAViewController.swift (hunk 4 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeAppUI/ReusableDetachedControllers/TwoFA/TwoFAViewController.swift]: ok

File: SafeUIKit/SafeUIKit/QRCodeScanner/ScannerViewController.swift (hunk 6 above)
Comment (DmitryBespalov) [reply SafeUIKit/SafeUIKit/QRCodeScanner/ScannerViewController.swift:72]: 😎

**Review by sche — APPROVED** [review]

**Pull request merged**